src/Sulu/Bundle/AdminBundle/Resources/js/components/CharacterCounter/tests/__snapshots__/CharacterCounter.test.js.snap

```diff
@@ -5,9 +5,7 @@ exports[`Should show a negative count 1`] = `
   <label
     class="characterCounter exceeded"
   >
-    -8
-     
-    sulu_admin.characters_left
+    -8 sulu_admin.characters_left
```

Comment:
What do you mean? Doesn't it look correct like this or am I missing something? 🤔

Reply:
all fine just waiting for the CI :)